src/components/home/experts-feed/experts-carousel.tsx

```diff
@@ -39,87 +47,113 @@ export const ExpertsCarousel = () => {
+              locations,
+              services,
+              id,
+              fullname,
+              average_rate,
+              owned_rates_count,
+              uuid,
+            },
+            index,
+          ) => {
+            const { expertServices, expertLocations } = formatUsersData(
+              locations,
+              services,
+              t("other.noLocation"),
+              t("other.noService"),
+            );
 
-            <div
-              className={cn(
-                "expert-card h-[184px] w-[298px] select-none rounded-3xl border border-blueMain bg-[linear-gradient(to_right,_#1a73e8,_#89c6ff)] p-3",
-                index % 2 === 1 &&
-                  "border-yellowMain bg-[linear-gradient(0.06deg,#F2B705_0.05%,#FFDB70_99.95%)]",
-              )}
-            >
-              <div className="y-[6px] mb-2 w-max rounded-3xl bg-[#FFFFFF33] px-[10px] text-[#FFFFFF99]">
-                {user.lastOnline}
-              </div>
+            return (
+              <CarouselItem
+                key={id}
+                className={"relative h-[184px] basis-[298px]"}
+              >
+                <Link
+                  href="#"
+                  className={
+                    "absolute right-[-8px] top-[-6px] flex h-10 w-10 items-center justify-center rounded-full border border-[#FFFFFF33] bg-blackMain transition-colors hover:bg-[#FFFFFF33]"
+                  }
+                >
+                  <Image
+                    src="/static/share.svg"
+                    alt="Share"
+                    width={20}
+                    height={20}
+                  />
+                </Link>
+                <Link href={`/expert/?uuid=${uuid}`}>
+                  <div
+                    className={cn(
+                      "expert-card h-[184px] w-[298px] select-none rounded-3xl border border-blueMain bg-[linear-gradient(to_right,_#1a73e8,_#89c6ff)] p-3",
+                      index % 2 === 1 &&
+                        "border-yellowMain bg-[linear-gradient(0.06deg,#F2B705_0.05%,#FFDB70_99.95%)]",
+                    )}
+                  >
+                    <div className="y-[6px] mb-2 w-max rounded-3xl bg-[#FFFFFF33] px-[10px] text-[#FFFFFF99]">
+                      {USER_ACTIVITY}
+                    </div>
 
-              <div className="select-none">
-                <div className="flex gap-2">
-                  <div className="flex h-14 w-14 justify-center rounded-full bg-white">
-                    <Image
-                      src={user.avatar}
-                      alt="Avatar"
-                      width={50}
-                      height={50}
-                    />
-                  </div>
+                    <div className="select-none">
+                      <div className="flex gap-2">
+                        <div className="flex h-14 w-14 justify-center rounded-full bg-white">
+                          <Image
+                            src={AVATAR}
```

Comment:
I think `UserSearchOut` should return some optional avatar, and if none provided, then we use default one

Reply:
This is in the plans when we create this field in the user model.

Reply:
On